Source/EventFlow.EntityFramework/EventFlow.EntityFramework.csproj

```diff
@@ -1,15 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="../Common.props" />
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net7.0;net8.0;net9.0</TargetFrameworks>
```

Comment:
Remove 7 and 9. Really try to only have the LTS versions. Keeping up with the non-LTS versions would be too much for me alone.